src/benchmark.cc

```diff
@@ -493,6 +504,15 @@ void BenchmarkImp::RangePair(int lo1, int hi1, int lo2, int hi2) {
   }
 }
 
+void BenchmarkImp::MinTime(double t) {
+  CHECK(t > 0.0);
```

Comment:
How long does the benchmark run for then? The maximum number of iterations? I think setting it to 0.0 is likely a bug and not a feature.